layer/CMakeLists.txt

```diff
@@ -45,14 +45,22 @@ endif()
 
 add_library(VkLayer_gfxreconstruct SHARED "")
 
+if (WIN32)
+    if(OPENXR_SUPPORT_ENABLED)
+        set(VK_LAYER_DEF_FILE "trace_layer_with_xr.def")
+    else()
+        set(VK_LAYER_DEF_FILE "trace_layer.def")
+    endif()
+endif()
+
```

Comment:
This is no longer needed.  Just add the 2 openxr entrypoints to the existing ones.  This code is actually being moved elsewhere in the dev branch with an upcoming PR #2098.

Reply:
Added a TODO.